JobConfig/primary/photon_conversion_map.fcl

```diff
@@ -0,0 +1,53 @@
+# -*- mode: tcl -*-
```

Comment:
What is the intended use of the output of this script?  The name suggests it is not a general-purpose primary.

Reply:
I'm sorry, I don't really know the purpose of each directory in JobConfig. The intent is to generate external RMC photons and write out the photon conversion map. This map is then read and re-sampled using the RMC_conversion_resampling.fcl configuration, which produces e+e- pairs.